dbt/adapters/duckdb/plugins/excel.py

```diff
@@ -7,9 +11,22 @@
 
 
 class Plugin(BasePlugin):
+    def initialize(self, plugin_config: Dict[str, Any]):
+        # Pass s3 settings to plugin environment
+        if 's3_access_key_id' in plugin_config:
```

Comment:
linting error that these and other string literals should be quoted with `"` instead of `'`

Reply:
My bad! Replaced them